autoplex/auto/rss/jobs.py

```diff
@@ -0,0 +1,689 @@
+            scalar_exp_pressure=scalar_exp_pressure,
+            scalar_pressure_exponential_width=scalar_pressure_exponential_width,
+            scalar_pressure_low=scalar_pressure_low,
+            scalar_pressure_high=scalar_pressure_high,
+            max_steps=max_steps,
+            force_tol=force_tol,
+            stress_tol=stress_tol,
+            hookean_repul=hookean_repul,
+            hookean_paras=hookean_paras,
+            keep_symmetry=keep_symmetry,
+            write_traj=write_traj,
+            num_processes_rss=num_processes_rss,
+            device=device_for_rss,
+            num_groups=num_groups,
+            config_type=config_type,
+        )
+        do_data_sampling = sample_data(
+            selection_method=rss_selection_method,
+            num_of_selection=num_of_rss_selected_structs,
+            bcur_params=bcur_params,
+            traj_path=do_rss.output,
+            random_seed=random_seed,
+            isolated_atom_energies=input["isolated_atom_energies"],
+        )
+        do_dft_static = DFTStaticLabelling(
+            e0_spin=e0_spin,
+            isolatedatom_box=isolatedatom_box,
+            isolated_atom=include_isolated_atom,
+            dimer=include_dimer,
+            dimer_box=dimer_box,
+            dimer_range=dimer_range,
+            dimer_num=dimer_num,
+            custom_incar=custom_incar,
+            custom_potcar=custom_potcar,
+        ).make(structures=do_data_sampling.output, config_type=config_type)
+        do_data_collection = collect_dft_data(
+            vasp_ref_file=vasp_ref_file,
+            rss_group=rss_group,
+            vasp_dirs=do_dft_static.output,
+        )
+        do_data_preprocessing = preprocess_data(
+            test_ratio=test_ratio,
+            regularization=regularization,
+            retain_existing_sigma=retain_existing_sigma,
+            scheme=scheme,
+            distillation=distillation,
+            force_max=force_max,
+            force_label=force_label,
+            vasp_ref_dir=do_data_collection.output["vasp_ref_dir"],
+            pre_database_dir=input["pre_database_dir"],
+            reg_minmax=reg_minmax,
+            isolated_atom_energies=input["isolated_atom_energies"],
+        )
+        do_mlip_fit = MLIPFitMaker(
+            mlip_type=mlip_type,
+            ref_energy_name=ref_energy_name,
+            ref_force_name=ref_force_name,
+            ref_virial_name=ref_virial_name,
+        ).make(
+            database_dir=do_data_preprocessing.output,
+            isolated_atom_energies=input["isolated_atom_energies"],
+            num_processes_fit=num_processes_fit,
+            apply_data_preprocessing=False,
+            auto_delta=auto_delta,
+            glue_xml=False,
+            device=device_for_fitting,
+            **fit_kwargs,
+        )
+
+        kt = current_kt - 0.1 if (current_kt - 0.1) > 0.1 else 0.1
+        current_iter += 1
+        if include_isolated_atom:
```

Comment:
Why is this arg overwritten ?

Reply:
During the iteration process, the iteration count should be updated, and the value of kt should also change to sample structures with different energy levels.